README.md

```diff
@@ -35,8 +35,12 @@ To download the model weights and tokenizer:
 1. Visit the [Meta Llama website](https://llama.meta.com/llama-downloads/).
 2. Read and accept the license.
 3. Once your request is approved you will receive a signed URL via email.
-4. Install the [Llama CLI](https://github.com/meta-llama/llama-stack): `pip install llama-toolchain`
-5. Run `llama model list` to determine the model ID you wish to download
+4. Install the [Llama CLI](https://github.com/meta-llama/llama-stack): `pip install llama-toolchain`.
+5. Run `llama model list` to show the latest available models and determine the model ID you wish to download.
+
+    **NOTE** 
```

Comment:
this feels a bit oddly formatted. can you just add it to the sentence itself? 

`determine the model ID you wish to download. *NOTE:* use `llama model list --show-all` if you want to see all available models.`